### PR TITLE
Feature/rampup command

### PR DESCRIPTION
**Pull request by SgtSilvio**

### CODE REVIEW

**Comment by SgtSilvio** [comment]
PLEASE DO NOT MERGE THIS PULL REQUEST BEFORE #70 IS MERGED
THE BASE BRANCH MUST BE CHANGED TO master

File: src/main/java/org/varroa/chunk/command/Command.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
         DISCONNECT,
         FOR,
         PUBLISH,
+        RAMPUP,
```
Comment (sauroter) [inline src/main/java/org/varroa/chunk/command/Command.java]: vll. RAMP_UP. RAMPUP sounds like a small goat

Comment (SgtSilvio) [reply src/main/java/org/varroa/chunk/command/Command.java]: should the command be renamed to `rampUp` (`RampUpCommand`, ...) as well?

Comment (SgtSilvio) [reply src/main/java/org/varroa/chunk/command/Command.java]: done

Comment (sauroter) [reply src/main/java/org/varroa/chunk/command/Command.java]: 👍

**Pull request merged**